lib/foreman_rh_cloud/engine.rb

```diff
@@ -79,7 +80,6 @@ class Engine < ::Rails::Engine
 
       ::Katello::UINotifications::Subscriptions::ManifestImportSuccess.include ForemanInventoryUpload::Notifications::ManifestImportSuccessNotificationOverride if defined?(Katello)
 
-      ::Host::Managed.include RhCloudHost
```

Comment:
Big thanks for the help on this @tbrisker ! :1st_place_medal: 
Looks like it wasn't needed and caused that `NameError (undefined local variable or method 'taxonomy_conditions' for#<Host::Managed::ActiveRecord_Relation:0x00007f0eaff32720>)`